portfolio-alex.html

```diff
@@ -0,0 +1,56 @@
+<!doctype html>
+
+<html>
```

Comment:
It's a good idea to specify the language.

```suggestion
<html lang="en">
```